api/envoy/config/endpoint/v3/load_report.proto

```diff
@@ -17,11 +17,11 @@ option java_outer_classname = "LoadReportProto";
 option java_multiple_files = true;
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
-// These are stats Envoy reports to GLB every so often. Report frequency is
-// defined by
+// [#protodoc-title: Load Report]
+
+// These are stats Envoy reports to GLB every so often. Report frequency is defined by
```

Comment:
I would reword this as 'These are stats Envoy reports to GLB at a frequency defined by...'